src/csharp/Grpc.Core.Tests/project.json

```diff
@@ -14,10 +14,10 @@
         },
         "copyToOutput": {
           "mappings": {
-            "nativelibs/windows_x64/grpc_csharp_ext.dll": "../../../vsprojects/x64/Debug/grpc_csharp_ext.dll",
-            "nativelibs/windows_x86/grpc_csharp_ext.dll": "../../../vsprojects/Debug/grpc_csharp_ext.dll",
-            "nativelibs/linux_x64/libgrpc_csharp_ext.so": "../../../libs/dbg/libgrpc_csharp_ext.so",
-            "nativelibs/macosx_x64/libgrpc_csharp_ext.dylib": "../../../libs/dbg/libgrpc_csharp_ext.dylib"
+            "grpc_csharp_ext.x64.dll": "../../../vsprojects/x64/Debug/grpc_csharp_ext.dll",
```

Comment:
btw, this file is generated using a mako template
(e.g. you need to update the template and run tools/buildgen/generate_project.sh - otherwise the sanity test will fail to indicate that)

Feel free to look at my PR https://github.com/grpc/grpc/pull/7295  (I am updating templates/src/csharp/build_options.include there).